packages/now-build-utils/src/detect-builders.ts

```diff
@@ -48,13 +53,50 @@ function hasBuildScript(pkg: PackageJson | undefined) {
   return Boolean(scripts && scripts['build']);
 }
 
-async function detectBuilder(pkg: PackageJson): Promise<Builder> {
-  for (const [dependency, builder] of getBuilders()) {
+function getFunctionBuilder(
+  file: string,
+  fallback: Builder | void,
+  { functions = {} }: Options
+) {
+  const key = Object.keys(functions).find(
+    k => minimatch(file, k) || file === k
+  );
+  const fn = key ? functions[key] : undefined;
+
+  if (!fn || (!fn.runtime && !fallback)) {
+    return fallback;
```

Comment:
At this point, `fallback` is always undefined. Is that correct?

Reply:
No, fallback could exist if only `fn` is `undefined`.